plugins/MantisGraph/MantisGraph.php

```diff
@@ -113,6 +113,7 @@ function resources() {
 			html_javascript_link( 'chart-' . CHARTJS_VERSION . '.min.js' );
 			html_javascript_link( 'chart.bundle-' . CHARTJS_VERSION . '.min.js' );
 		}
+		html_javascript_link( '../plugins/MantisGraph/MantisGraph.js' );
```

Comment:
Should chart js libs, ("chart-xxxx.min.js", etc) also be provided by the plugin?